ledger/apply/application_test.go

```diff
@@ -571,9 +564,6 @@ func TestAppCallApplyCreate(t *testing.T) {
 	a.Zero(b.putAppParams)
 	// ensure original balance record in the mock was not changed
 	// this ensure proper cloning and any in-intended in-memory modifications
-	//
-	// known artefact of cloning AppLocalState even with empty update, nil map vs empty map
-	saved.AppLocalStates = map[basics.AppIndex]basics.AppLocalState{}
```

Comment:
This appeared to be testing an artifact of the mock, not of a real ledger. The artifact is gone now, because `maps.Clone` maintains `nil`.